docker/rumble/client_scripts/rumble_client.py

```diff
@@ -0,0 +1,83 @@
+"""
```

Comment:
Every python script should start with the following statement (choose python or python3)
```
#!/usr/bin/env python
```
This statement allows choice of proper python version and allow to run script as `/path/script.py`, i.e. withiout python command in any shell. Also make sure you change permission of the python file to have executable bit